types/graphql-list-fields/package.json

```
@@ -6,7 +6,7 @@
         "https://github.com/jakepusateri/graphql-list-fields#readme"
     ],
     "dependencies": {
-        "graphql": "^14.5.3"
+        "graphql": "*"
```

Personally, I would pin this same version the library uses: <https://github.com/jakepusateri/graphql-list-fields/blob/8324e55c990638dd84d74f01e585a54f394424ef/package.json#L30>

```suggestion
        "graphql": "^16.8.1"
```

This way, if there ever does happen to be a breaking change, the types won't end up on a newer version than the library.

---

That was what I was initially going to do, but I did a search for similar PRs for an idea of what I should be doing and [found this](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64366#discussion_r1136260631) so decided on the same thing. 

If you'd like, I'm more than happy to change it to the specific version number.

---

Now that I think about it it's probably fine either way, since the `*` should just let it match whatever version the main package depends on